openmetadata-ui/src/main/resources/ui/src/utils/DataQuality/TestSummaryGraphUtils.ts

```diff
@@ -87,9 +87,14 @@ export const prepareChartData = ({
 
   dataPoints.reverse();
 
+  // get params from the result
+  const testCaseResultParams = testCaseResults.find(
+    (result) => result.testResultValue?.length
+  );
+
```

Comment:
@ShaileshParmar11 , this will return the first test case result that has a non-empty testResultValue. so if we have
```js
const testCaseResults = [
  { testResultValue: null },
  { testResultValue: [] },
  { testResultValue: [1, 2, 3] },
  { testResultValue: [4, 5] }
]; 
```

Then it will return the 3rd object and ignore the 4th one, is that intentional?

Reply:
all the values in the test result will have the same parameter except when the test is aborted! so either it will be empty or have same length of value!